features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -278,7 +278,14 @@
 
     <feature name="openhab-binding-groheondus" description="GROHE ONDUS Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.groheondus/${project.version}</bundle>
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">wrap:mvn:com.fasterxml.jackson.core/jackson-core/2.9.6</bundle>
```

Comment:
I'm abslutely unsure if that's correct. From my personal feeling, I would say I did something terribly wrong: It's kind of useless to use a dependency management system, if I, as a user of a third-party library, need to specific the dependencies of _this_ third party library. This should be taken care of in the dependency managemtn itself :/

Reply:
Read: Is there a way of instructing karaf something like "please look at the pom dependencies and just wrap all of them as dependencies in the OSGi container"?

Reply:
That's correct. The karaf container loads those dependency during runtime. Independent of compile time dependencies.

In the OSGi world you are usually compiling against an -api bundle and the container later on loads the -impl dependency.

Reply:
And yes it is possible to generate the karaf feature file automatically from the pom. I have already opened an issue.